### PR TITLE
Remove Travis test setup

### PR DESCRIPTION
**Pull request by tkarna**
Remove Travis test setup that's now obsolete.

### CODE REVIEW

File: test/conftest.py [hunk 1]
```diff
@@ -6,12 +6,6 @@ def pytest_addoption(parser):
                      help="Only run tests marked for Travis")
 
 
-def pytest_runtest_setup(item):
-    not_travis = item.get_marker("not_travis")
-    if not_travis is not None and item.config.getoption("--travis"):
-        pytest.skip("Skipping test marked not for Travis")
```
Comment (tkarna) [inline test/conftest.py]: @wence- I guess we can now remove all Travis related bits in `conftest.py`? Do we still need the `still alive` hack?

Comment (wence-) [reply test/conftest.py]: No

Comment (tkarna) [reply test/conftest.py]: Okay, I've cleaned up `conftest.py` some more.

**Comment by tkarna** [comment]
I've cleaned up `conftest.py`. Any thoughts?

**Comment by tkarna** [comment]
Is this good to go?

**Pull request merged**